plugins/woocommerce-admin/client/customize-store/assembler-hub/hooks/auto-block-preview-event-listener.ts

```diff
@@ -175,9 +175,10 @@ const addInertToAssemblerPatterns = (
 const addInertToAllInnerBlocks = ( documentElement: HTMLElement ) => {
 	const body = documentElement.ownerDocument.body;
 	const observerChildList = new window.MutationObserver( () => {
-		const parentBlocks = body.getElementsByClassName(
-			'block-editor-block-list__layout'
-		)[ 0 ].children;
+		const parentBlocks =
+			body.getElementsByClassName(
+				'block-editor-block-list__layout'
+			)[ 0 ]?.children ?? [];
```

Comment:
This change is not related to this PR, but I noticed that in some cases, the browser threw an error because `body.getElementsByClassName('block-editor-block-list__layout')[ 0 ]` returned `undefined`